src/NUnitTestAdapter/NUnitTestAdapter.cs

```diff
@@ -100,37 +92,41 @@ public static bool IsRunningUnderIDE
         // each Discover or Execute method must call this method.
         protected virtual void Initialize(IDiscoveryContext context, IMessageLogger messageLogger)
         {
-            Settings.Load(context);
-
-            //var settingsProvider = runSettings.GetSettings(SettingsName) as NUnitAdapterSettingsProvider;
-            //var settings = settingsProvider != null ? settingsProvider.Settings : new NUnitAdapterSettings(); 
-            
             try
             {
-                var registry = RegistryCurrentUser.OpenRegistryCurrentUser(@"Software\nunit.org\VSAdapter");
-                UseVsKeepEngineRunning = registry.Exist("UseVsKeepEngineRunning") && (registry.Read<int>("UseVsKeepEngineRunning") == 1);
-                ShadowCopy = registry.Exist("ShadowCopy") && (registry.Read<int>("ShadowCopy") == 1);
-#if DEBUG && VERBOSE
-                Verbosity = 1;
-#else
-                Verbosity = (registry.Exist("Verbosity")) ? registry.Read<int>("Verbosity") : 0;
-#endif
+                Settings.Load(context);
             }
             catch (Exception e)
             {
-                messageLogger.SendMessage(TestMessageLevel.Error, "Unable to access registry. Default settings will be used");
+                messageLogger.SendMessage(TestMessageLevel.Error, "Error initializing RunSettings. Default settings will be used");
```

Comment:
We should try to give some more specific information here.  Just "Error initializing" doesn't tell the user what is possibly wrong.